Payments/Operation/01_ManageAuthorizations.md

```diff
@@ -0,0 +1,126 @@
+
+If everything works fine with the posting of a payment, you do not need to monitor the *Authorizations* menu entry. In case of errors you can capture, cancel, or void an authorization transaction manually.   
+> [Info] You can only change the status of an authorization transaction, but not any data.
+
+
+## Check authorization transaction
+An authorization transaction provides numerous payment-related details, which may vary depending on the payment service provider settings. To decide, whether you can manually capture, cancel, or void an authorization transaction, you need to check its details.
+
+#### Prerequisites
+
+- At least one connection has been created, see [Create a connection](../Integration/01_ManageConnections.md#create-a-connection).
+- At least one authorization transaction has been created.
+- It is recommended that you have added the *Status ID* column to your view so that you are able to sort or filter for specific statuses. 
+
+#### Procedure
+
+*Payments > Authorizations > Tab LIST*
+![Authorizations](../../Assets/Screenshots/Payments/Authorizations/LISTAuthorizations.png "[Authorizations]")
+
+1. Check the Authorization transaction status in the *Status* column. The status displays the current stage in the payment process. You can use the status ID prefixed below to filter the list. The following statuses are available: 
+    - **1 - Preparing**  
+        The transaction has been created, but is not yet transferred.
+    - **2 - Unconfirmed**   
+        A refund is reserved but not yet released by an Actindo user. This status is relevant for some functions only, for example, the cancellations and chargebacks.
+    - **3 - Success**  
+        Actindo has got the response from the service payment provider, that the transfer was successfully.
+    - **4 - Failure**   
+        The payment service provider has received the request, but it cannot process it. If available for the connection, the status information field provides further information.
+    - **5 - Error**   
+       Errors have been occurred during transfer.
+    - **6 - Void**   
+       The transaction has been voided, see [Void authorization transaction](01_ManageAuthorizations.md#void-authorization-transaction).
+    
+2. Click on the authorization transaction to see the details.   
+    The sub-tabs of the transaction are opened.   
+     
+    *Payments > Authorizations > Tab LIST > Select Authorization transaction*  
+
+    ![Authorization attributes and logs](../../Assets/Screenshots/Payments/Authorizations/CheckAttributes.png "[Authorization attributes and logs]")
+
+3. Check the attributes and logs of the transaction. For detailed information, see the following:
+   - [Authorizations &ndash; Attributes](../UserInterface/01a_ListAuthorizations.md#authorizations-–-attributes)
+   - [Authorizations &ndash; Logs](../UserInterface/01a_ListAuthorizations.md#authorizations-–-logs)
```

Comment:
Der Links zu UI (List Authorization) ist schon oben aufgeführt. Das kommt nachher in Related content auf der Webseite. 
Hier geht es eher darum, was der Kunde hier checken kann/soll, um eine Entscheidung zu treffen. Welche Details von allen Verfügbaren kann der Kunde prüfen (welche sind interessant)?
Als Idee kannst du bei Fulfillment gucken: https://github.com/Actindo-AG/Documentation/blob/main/Fulfillment/Operation/02_CheckShipment.md

Auf jeden Fall als Procedure weiter beschreiben und, wenn er beide Reiter anklicken und prüfen soll, in separaten Schritten (und mit separaten Screenshots) beschreiben.

Reply:
Es ist vielleicht ein bisschen überdesignt, dafür ein eigenes Topic anzulegen. Procedure beschrieben